models/event_argument_extraction/Linker.py

```diff
@@ -0,0 +1,45 @@
+from EntLink.api import init, Related, filter
+import spacy
+
+class Linker(object):
+    def __init__(self):
+        self.path_to_ckpt = ''
+        self.tokenizer = None
+        self.model = None
+        self.model_para = dict()
```

Comment:
```suggestion
        self.model = self.instantiate()
```

Reply:
it is better to keep stuff in the constructor simple and only define attributes that are shared within class here

Reply:
done